marlowe-runtime/contract/Language/Marlowe/Runtime/Contract.hs

```diff
@@ -6,46 +6,106 @@
 module Language.Marlowe.Runtime.Contract
```

Comment:
Changes in this file:

- Move server sources to component outputs
- Move otel rendering from Logging.hs